### PR TITLE
Clarify version to use with Rails 6.0

### PR DESCRIPTION
**Pull request by GCorbel**
By reading the README, it wasn't clear to me what version I'm supposed to use with Rails 6.0 and, to me "Rails from 5.0 to 6.0" means Rails 6.0 is included. After I read [versioning-strategy](https://github.com/rspec/rspec-rails/blob/main/rfcs/versioning-strategy.md), I understand rspec-rails 5.x have to used for Rails 6 and above. Also, the reasons of this rule wasn't clear.

I also changed the formatting.

Let me know if I misterstood.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,9 +8,7 @@ They’re also specifications (or _specs,_ for short):
 detailed explanations of how the application is supposed to behave,
 expressed in plain English.
 
-Use **[`rspec-rails` 4.x][]** for Rails from 5.0 to 6.0.
-Use **[`rspec-rails` 3.x][]** for Rails earlier than 5.0.
-Use **[`rspec-rails` 1.x][]** for Rails 2.x.
+According to [RSpec Rails new versioning strategy][], use **[`rspec-rails` 5.x][]** for Rails 6.x, **[`rspec-rails` 4.x][]** for Rails 5.x, **[`rspec-rails` 3.x][]** for Rails earlier than 5.0 and **[`rspec-rails` 1.x][]** for Rails 2.x.
```
Comment (JonRowe) [inline README.md:11]:
```suggestion
According to [RSpec Rails new versioning strategy][] use:
* **[`rspec-rails` 5.x][]** for Rails 6.x.
* **[`rspec-rails` 4.x][]** for Rails from 5.x or 6.x.
* **[`rspec-rails` 3.x][]** for Rails earlier than 5.0.
* **[`rspec-rails` 1.x][]** for Rails 2.x.
```

**Comment by benoittgt** [comment]
Thanks @GCorbel ;)
